services_app/build.gradle

```diff
@@ -30,10 +30,11 @@ android {
         applicationId(groupId + servicesName)
         minSdkVersion(minVersion)
         targetSdkVersion(targetVersion)
-        versionCode(releaseVersionCode)
```

Comment:
again you should revert this change since you are changing the code to not match the style that exists